packages/docs/components/Playground.tsx

```diff
@@ -1,6 +1,6 @@
 import { Application, coreNodeProvider, Diagram } from '@data-story/core';
 import React from 'react';
-import { DataStory } from '@data-story/ui';
+import { AllotmentDataStory } from '@data-story/ui';
```

Comment:
Since AllotmentDataStory will now be used as the new root component (?). for simplicity of the end user, can we consider keeping the root component name "DataStory" and providing the alotment feature anyway? Not sure how that is achieved best, maybe via a flag or, forwarding to subcomponents, or something else. I think we can assume usage of the alotment will be the default/most common usage. As such, can we have that reflect in that user only need to use the name <DataStory ... /> and not the good, but very explicit <AllotmentDataStory ... /> ?